sql/core/src/test/scala/org/apache/spark/sql/execution/python/PythonUDFSuite.scala

```diff
@@ -28,6 +28,8 @@ class PythonUDFSuite extends QueryTest with SharedSparkSession {
 
   val scalaTestUDF = TestScalaUDF(name = "scalaUDF")
   val pythonTestUDF = TestPythonUDF(name = "pyUDF")
+  assume(pythonExec.equals("python3"))
```

Comment:
If this is the issue, assert that it's either python or python3? the python 3 binary could be just python.

Reply:
Since we drop Python2 officially, I'm looking this problem in more broader perspective.

Reply:
Technically, we hit the similar issue before at SPARK-32422 two days ago. cc @HyukjinKwon .
```
[SPARK-32422][SQL][TESTS] Use python3 executable instead of python3.6 in IntegratedUDFTestUtils
```
```
  lazy val pythonExec: String = {
    val pythonExec = sys.env.getOrElse(
      "PYSPARK_DRIVER_PYTHON", sys.env.getOrElse("PYSPARK_PYTHON", "python3"))
    if (TestUtils.testCommandAvailable(pythonExec)) {
      pythonExec
    } else {
      "python"
    }
  }
```

At SPARK-32422, `python3.6` doesn't work correctly and @HyukjinKwon replaced it from `python3.6` to `python3`. The root cause is that we have multiple Python installation environment.

Reply:
For now, let me override this simply.